Add redirect for soil-dsi-bridge ontology

GuaPi-D opened the pull request:
This pull request adds a redirect for https://w3id.org/projects/soil-dsi-bridge
pointing to https://guapi-d.github.io/soil-dsi-bridge/soil-dsi-bridge.ttl

dgarijo commented:
Please add a readme file indicating who will maintain this w3id

davidlehn commented:
Why are you using the `/projects/` prefix?  Would you prefer a shorter URL like `https://w3id.org/soil-dsi-bridge`?  I'm not sure the generic `projects` name should be claimed and used as only a parent directory like this.

GuaPi-D commented:
Thank you for the feedback! I have removed the `/projects/soil-dsi-bridge` directory and added a new `/soil-dsi-bridge` directory as suggested. The `.htaccess` and `README.md` files have been updated accordingly.
